stat/distuv/triangle.go

```diff
@@ -0,0 +1,177 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math"
+	"math/rand"
+)
+
+// UnitTriangle is an instantiation of the triangle distribution with A=0, C=0.5, and B=1
+var UnitTriangle = Triangle{A: 0, C: 0.5, B: 1}
+
+// Triangle represents a triangle distribution (https://en.wikipedia.org/wiki/Triangular_distribution).
```

Comment:
There are constraints on a, b and c (-inf < a < inf, a < b; a <= c <= b) that are not mentioned here and are never checked for. It seems to me that this should have a `NewTriangle(a, b, c float64, rnd *rand.Rand) Triangle` that does this check (and also in the parameter marshaling), and the fields should be unexported.